app/assets/javascripts/models/user.js

```diff
@@ -1,3 +1,15 @@
 module.exports = Backbone.Model.extend({
-  name: 'user'
+  name: 'user',
+  initialize: function() {
+  },
+  getCurrent: function(callback) {
+    var options = {
+      type: 'GET',
+      dataType: 'json',
+      url: '/users/current',
+      success: callback,
+    };
+
+    return $.ajax(options);
```

Comment:
Isn't it supposed to return a user instance? Why not add a _static_ method that: does the ajax, creates a new user instance, and then return it in a promise? There's no reason to create a user instance and then call a `getCurrent` on it to return another user, it's counterintuitive. We could do something like this:

```js
var User = module.exports = Backbone.Model.extend({
    name: 'user'
  },
  { // << notice that this is the _second_ parameter of `extend`, that receives the static methods
    getCurrent: function(callback) {
      var options = {
        type: 'GET',
        dataType: 'json',
        url: '/users/current'
    };

    return $.ajax(options).then(function(userData) {
      return new User(userData);
    });
  }
});

// Then you'd do something like this:

User.getCurrent().then(getCurrentUserSuccess);
```